app/models/activity.rb

```diff
@@ -7,6 +7,8 @@ class Activity < ActiveRecord::Base
 
   validates :content, :title, :team, presence: { if: ->(act) { act.kind == 'status_update' } }
 
+  delegate :students, to: :team
```

Comment:
@carpodaster, @F3PiX is just beginning to learn about testing, so we won't be able to do this any time soon. would it be ok if we skip it for now?